model/client.go

```diff
@@ -239,7 +247,7 @@ func (c *Client) GetImportStatus(importID string) (*ImportStatus, error) {
 // closeBody ensures the Body of an http.Response is properly closed.
 func closeBody(r *http.Response) {
 	if r.Body != nil {
-		_, _ = ioutil.ReadAll(r.Body)
+		_, _ = io.ReadAll(r.Body)
```

Comment:
```suggestion
		_, _ = io.Copy(io.Discard, r.Body)
```

This may only be a minor performance increase.